FormData.js

```diff
@@ -159,6 +162,9 @@ class FormDataPolyfill {
    * @return  {Undefined}
    */
   set(name, value, filename) {
+    if (!(value instanceof File) && !(value instanceof Blob) && typeof value != 'string') {
+      value += ''
+    }
```

Comment:
File is also a instance of Blob so that can be reduced down to just one... but how about just doing 
```js
wm(this)[name + ''] = [[value instanceof Blob ? value : value + '', filename]]
```